hydrus/data/crud.py

```diff
@@ -56,6 +56,10 @@ def insert(object_, id_=None):
     """Insert an object to database [POST] and returns the inserted object."""
     # NOTE: We are inserting the object, no need to check if similar one already exists.
     #       Data can be redundant/identical, they must have different "@id"
+
+    # TODO: @chrizandr You're not setting instance.type_ and you're not using keymap
+    # to dereference class types. Try any other category other then thermal for insertion
+    # Also, make sure crud.py passes all the tests in test_db.py (Currently all failing)

```

Comment:
You're partially right. We need to tell which class an instance belongs to but I don't think there's a need to use a `GraphIAC` triple to store that. Also we have `GraphIII` and `GraphIIT` where where we'll have to define which class every instance belongs to. We already have a system defined for this in the Instance table.
```python
class Instance(Base):
    """Model for Object/Resource.
    Instances are instances of some kind/classes that are served through the API.
    """

    __tablename__ = "instances"

    id = Column(Integer, primary_key=True)
    name = Column(String)
    type_ = Column(Integer, ForeignKey("classes.id"), nullable=True)
```
The `type_` attribute can be used to define which class any instance belongs to. What say?